build-tools/android-toolchain/android-toolchain.projitems

```diff
@@ -69,6 +69,21 @@
       <HostOS></HostOS>
       <DestDir>platforms\android-10</DestDir>
     </AndroidSdkItem>
+
+    <!-- Note: if the version is changed here (which forces the change of path), make sure to update Configure.props! -->
```

Comment:
Given that we're now requiring that Mono install this, why have these these entries at all for Darwin/Linux?

Reply:
For completness, to match the other sets. But I can remove them, no problem

Reply:
Removed Linux/Darwing, GitHub missed the change